packages/core/admin/admin/src/pages/HomePage/CloudBox.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { useTracking } from '@strapi/helper-plugin';
+import { Box, Flex, Stack, Typography } from '@strapi/design-system';
+import cloudIconBackground from './assets/strapi-cloud-background.png';
+import cloudIcon from './assets/strapi-cloud-icon.svg';
+import cloudFlags from './assets/strapi-cloud-flags.svg';
+
+const BlockLink = styled.a`
+  text-decoration: none;
+`;
+
+const CloudCustomWrapper = styled.div`
+  width: 56px;
+  height: 56px;
+  background-image: url(${({ src }) => src});
+  padding: 11px;
+  margin-right: ${({ theme }) => theme.spaces[6]};
+`;
+
+const CloudIconWrapper = styled.div`
+  background: rgba(255, 255, 255, 0.3);
+  border-radius: 4px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+`;
```

Comment:
Same here. This is better written as:

```suggestion
const CloudIconWrapper = styled(Flex)`
  background: rgba(255, 255, 255, 0.3);
`;
```

<CloudIconWrapper hasRadius justiftContent="center" alignItems="center" height="100%" />

Could you try and get rid of the background color-value too? We usually rely on theme values, because they work seamlessly in light and darkmode.

We are trying to use props over custom CSS wherever possible.

Reply:
In this case I'm not sure the image would change when the theme changes

Reply:
Me neither, but maybe worth checking :)

Reply:
In this case you can leave it. 👍🏼  Just wanted to double-check.